sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -263,6 +266,23 @@ builder.Services.AddOpenTelemetry().UseAzureMonitor(o =>
 });
 ```
 
+#### Drop a Metrics Instrument
+
+If you want to exclude specific instruments from being collected in your application's telemetry, such as metrics from the distro or custom metrics, use the following code snippet. 
+
+```C#
+builder.Services.ConfigureOpenTelemetryMeterProvider(b =>
+    b.AddView((instrument) =>
+    {
+        if (instrument.Name == "http.server.request.duration")
+        {
+            return null;
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/metrics/customizing-the-sdk#drop-an-instrument

Reply:
Cijo's example shows returning `MetricStreamConfiguration.Drop`. I changed this is the Readme.

Cijo's example also shows matching on the Meter.Name. How would a user find the meter name? I don't see this in the dotnet doc.

Reply:
Use this API -  .AddView(instrumentName: "MyCounterDrop", MetricStreamConfiguration.Drop)